elephant/statistics.py

```diff
@@ -215,6 +215,13 @@ def lv(v):
     v : quantity array, numpy array or list
         Vector of consecutive time intervals
 
+    with_nans : bool, optional
+        If `True`, correlations of empty spike trains are given NaN values. If
+        `False`, a boolean array indicating empty (`False`) and non-empty (`True`)
```

Comment:
The boolean mask is not returned as far as I can see. Instead `np.NaN` is returned.